README.md

```diff
@@ -287,6 +287,100 @@ However, this will rebuild entire VM, which means that besides your own scripts,
 $ capstan package compose --update node-micro/tutorial
 ```
 
+## Deploying Unikernels on Kubernetes
+You can deploy `osv-microservice-demo` application on Kubernetes at ease. If you don't have a working Kubernetes cluster up-and-running with [Virtlet](https://github.com/Mirantis/virtlet) plugin installed,
+please follow these quick steps.
```

Comment:
Please add a comment that this has been tested on Ubuntu XX.YY; also invite people to follow some external docs on installing the tools.